pom.xml

```diff
@@ -57,8 +55,25 @@
         <version.arm-oraclelinux-wls-cluster-deletenode>1.0.1</version.arm-oraclelinux-wls-cluster-deletenode>
         <!--  versions  end -->
 
+        <git.tag>main</git.tag>
```

Comment:
I realize these properties were moved up from the VM offer to the top level module and they did not have detailed comments before.  Even so, I want them to have detailed comments now.